code/modules/antagonists/dodgeball/dodgeball.dm

```diff
@@ -61,14 +61,21 @@
 	W.item_flags |= NODROP
 	W.update_label(H.real_name)
 	H.equip_to_slot_or_del(W, SLOT_WEAR_ID)
-
 	mine = new(H)
 //	if(!GLOB.doomballer)
 //		sword.flags_1 |= ADMIN_SPAWNED_1 //To prevent announcing
 //	sword.pickup(H) //For the stun shielding
 	H.put_in_hands(mine)
 
 
+	var/obj/item/bloodcrawl/antiwelder = new(H)
```

Comment:
At this point, make a new item subtype to be used for both highlander and doomball instead of adding in more copypasta.